### PR TITLE
docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-an…

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Add platform suspend and resume test case description

Signed-off-by: Przemysław Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Dasharo compatibility: platform suspend and resume
```
Comment (damiankaras) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:1]: `compatibility` should start from a capital letter

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Test cases
+
+### SUSP001.001 Platform suspend and resume
+
+**Test description**
+
+This test verifies whether the DUT might be put into suspend mode and then, by
```
Comment (damiankaras) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:9]: `then by using`

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Dasharo compatibility: platform suspend and resume
+
+## Test cases
+
+### SUSP001.001 Platform suspend and resume
+
+**Test description**
+
+This test verifies whether the DUT might be put into suspend mode and then, by 
+using the power button, might be resume properly. This test case may be re-done
```
Comment (damiankaras) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:10]: `might be properly resumed`

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+1. Proceed with the
+   [Generic test setup: firmware](generic-test-setup#firmware)
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into OS.
+1. Run command `date` and write down output of the command.
+1. Run command `systemctl suspend -i`
```
Comment (damiankaras) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:29]: missing dot at the end of line

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+1. Control status of the platform.
+1. Press the power button once and note the result.
+1. Run the following command: `journalctl | grep suspend`.
+
+**Expected result**
+
+1. After running suspend command the platform should enter the suspend mode.
+2. After pressing power button the platform should initiate the resume
```
Comment (damiankaras) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:37]: we can stay with the "1. 1. 1." markdown enumeration convention

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+1. After running suspend command the platform should enter the suspend mode.
+2. After pressing power button the platform should initiate the resume 
+    procedure.
+3. Output of the last command should contains the line with the statement:
+
+            systemd[1]: systemd-suspend.service: Succeeded
+
+4. Date of the above-described event should be later than date from 
+    command `date` from point 3.
```
Comment (damiankaras) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:44]: maybe `the third` point?

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:1]: Improved

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md (hunk 6 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md:44]: I think that `the third point` sounds better. Improved

**Pull request merged**